vmdb/app/models/ems_openstack_infra.rb

```diff
@@ -16,7 +16,7 @@ def supports_port?
   end
 
   def supported_auth_types
-    %w(default amqp)
+    %w(default amqp keypair)
```

Comment:
Not sure on the auth_type name here.  Seems too generic.  Perhaps `ssh_something`? @blomquisg Thoughts?